lib/ddtrace/opentracer/distributed_headers.rb

```diff
@@ -34,6 +34,12 @@ def sampling_priority
         value
       end
 
+      def origin
+        hdr = @carrier[HTTP_HEADER_ORIGIN]
+        # Only return the value if it is not an empty string
+        hdr if hdr != ''
```

Comment:
Is `nil` an acceptable output here?

Reply:
Yeah, I want `nil` instead of `''` so I can tell whether we should set/propagate this header further.